packages/swapkit/helpers/src/modules/assetValue.ts

```diff
@@ -84,7 +84,7 @@ export class AssetValue extends BigIntArithmetics {
     return this.chain === chain && this.symbol === symbol;
   }
 
-  eq(assetValue: AssetValue) {
+  eq(assetValue: AssetValue): boolean {
```

Comment:
Let TS infer type here

Reply:
faced a type error cause of return type missing.